src/redux/products/slice.ts

```diff
@@ -33,20 +34,14 @@ const INITIAL_STATE: ItemsState = {
   allCategories: null,
   offset: 0,
   sortValue: SortValue.lowPrice,
-  selectedWeight: []
+  selectedWeight: [],
+  isFavorite: []
```

Comment:
did you add this to types?

Reply:
I'll add the types, this is a small preparation for the future